client/src/util/finiteExtent.ts

```diff
@@ -21,13 +24,14 @@ function finiteExtent(tarr: any) {
       break;
     }
   }
-  for (; i < tarr.length; i += 1) {
-    const val = tarr[i];
-    if (Number.isFinite(val)) {
-      if (min > val) min = val;
-      if (max < val) max = val;
+  if (min && max)
```

Comment:
TS compiler couldn't see that for loop would be skipped in the case that min and max weren't defined

Reply:
This introduces a serious bug -- if min/max are === 0, it won't do the second loop.

you want

`if (min === undefined & max === undefined) ...`